Backend/DataAccessLayer/Models/User.cs

```diff
@@ -39,6 +39,10 @@ public User()
         public string SecurityQ2Answer { get; set; }
         public string SecurityQ3 { get; set; }
         public string SecurityQ3Answer { get; set; }
+
+        [Required]
+        public int IncorrectPasswordCount { get; set; }
```

Comment:
If this is non-nullable, make sure the user constructor for this model sets a default value (0)